x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/classification_exploration/evaluate_panel.tsx

```diff
@@ -54,6 +54,43 @@ interface Props {
   searchQuery: ResultsSearchQuery;
 }
 
+enum SUBSET_TITLE {
+  TRAINING = 'training',
+  TESTING = 'testing',
+  ENTIRE = 'entire',
+}
+
+const entireDatasetHelpText = i18n.translate(
+  'xpack.ml.dataframe.analytics.classificationExploration.confusionMatrixEntireHelpText',
+  {
+    defaultMessage: 'Normalized confusion matrix for entire dataset',
+  }
+);
+
+const testingDatasetHelpText = i18n.translate(
+  'xpack.ml.dataframe.analytics.classificationExploration.confusionMatrixTestingHelpText',
+  {
+    defaultMessage: 'Normalized confusion matrix for testing dataset',
+  }
+);
+
+const trainingDatasetHelpText = i18n.translate(
+  'xpack.ml.dataframe.analytics.classificationExploration.confusionMatrixTrainingHelpText',
+  {
+    defaultMessage: 'Normalized confusion matrix for training dataset',
+  }
+);
+
+function getHelpText(dataSubsetTitle: string) {
+  let helpText = entireDatasetHelpText;
+  if (dataSubsetTitle === SUBSET_TITLE.TESTING) {
+    helpText = testingDatasetHelpText;
+  } else if (dataSubsetTitle === SUBSET_TITLE.TRAINING) {
+    helpText = trainingDatasetHelpText;
+  }
+  return helpText;
```

Comment:
Nit: Might be a rare case where a `switch` statement could make this more clear :)